dxr/config.py

```diff
@@ -182,6 +184,9 @@ def __init__(self, input, relative_to=None):
                     except SchemaError as exc:
                         raise ConfigError(exc.code, [section])
 
+        self._section['es_catalog_index'] = self._section['es_catalog_index'].format(
+            config_path_hash=self.path_hash())
```

Comment:
And since you didn't change the default values of the es_alias or es_catalog_index options, we should be able to deploy this without breaking anything. Do you agree? (Also .format() doesn't care if you give it extra args for which there aren't tokens in the string, so that's good too.)

Reply:
Yes, it was intended to be entirely backward compatible.  The change should be transparent for existing users.